### ISSUE TITLE
Support for React 18+

### ISSUE THREAD

**Issue by unidesigner** [Kanaries/graphic-walker#50]
Hi! I came across this very nice package, and I would like to integrate it into a platform for neuroscience research. Unfortunately, the platform requires 18. Are there any plans to update graphic-walker to support React 18? Thank you.

**Comment by ObservedObserver** [comment]
Sure, react 18 ecosystem is stable enough now; we will add support to it soon.

**Comment by ObservedObserver** [comment]
Hi, `graphic-walker` now supports react18+ (since v0.3.1), you can try it now.

For using react18 with graphic-walker, try not use the `strict mode` which will leads to drag and drop issue by `react-beautiful-dnd`.

**Comment by unidesigner** [comment]
Thanks for the quick response and update! I am looking forward to test the release as it becomes available via `npmjs`. Can you tell me what you mean with `strict mode`?

**Comment by ObservedObserver** [comment]
It is now available in npmjs: `@kanaries/graphic-walker@0.3.1`

For example, the entry of your react 18 code might look like 
```ts
ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
)
```

You need to remove the `<React.StrictMode>` for graphic-walker, and it should looks like:

```
ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(<App />)
```
